apps/server/src/modules/oauth/oauth.module.ts

```diff
@@ -0,0 +1,28 @@
+import { Configuration } from '@hpi-schul-cloud/commons/lib';
+import { HttpModule } from '@nestjs/axios';
+import { Module } from '@nestjs/common';
+import { SymetricKeyEncryptionService } from '@shared/infra/encryption';
+import { UserRepo } from '@shared/repo';
+import { SystemRepo } from '@shared/repo/system';
+import { Logger, LoggerModule } from '@src/core/logger';
+import { AuthorizationModule } from '../authorization';
+import { OauthSSOController } from './controller/oauth-sso.controller';
+import { OauthUc } from './uc/oauth.uc';
+
+const logger = new Logger();
+
+const key: string = Configuration.get('LDAP_PASSWORD_ENCRYPTION_KEY') as string;
```

Comment:
Line who produce the error at the e2e test

Reply:
fixed. thx. @mamutmk5 @Metauriel